build.sh

```diff
@@ -1,7 +1,7 @@
 #!/usr/bin/env bash
 
-# This file is part of MinIO Direct CSI
-# Copyright (c) 2021 MinIO, Inc.
+# This file is part of MinIO DirectPV
+# Copyright (c) 2022 MinIO, Inc.
```

Comment:
You should retain `2021` and previous years copyright. It should be like `2021, 2022`. It is applicable everywhere except files those are all appearing from `2022`.

Reply:
checked all PR merged in 2022. Didnt find any new files added.  modified to `2021, 2022` in all the files.

Reply:
Files added prior to 2022 should have retain previous years copyright and files added in 2022 shouldn't for obvious reason.